lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -20,7 +20,7 @@
 )
 from lib.execution_engine2.sdk.EE2Constants import ConciergeParams
 from lib.execution_engine2.utils.CondorTuples import CondorResources
-from lib.execution_engine2.utils.KafkaUtils import KafkaCreateJob, KafkaQueueChange
+from execution_engine2.utils.KafkaUtils import KafkaCreateJob, KafkaQueueChange
```

Comment:
?? why no `lib.`?

Reply:
I'm not sure why `lib` is in the path in the first place - the SDK sets the python path to `[repo]/lib` so it shouldn't be necessary. I'm a bit surprised it works at all. I imported without `lib` in my tests, and apparently when you have the same named tuple with different import lines either the mocks or the `Kafka*.__eq__` method gets confused and says tuples with identical contents aren't equal.

I'm gradually going around and removing the `lib`s when I need to.